projects/js-packages/publicize-components/src/components/share-status/share-info.tsx

```diff
@@ -0,0 +1,38 @@
+import { getDate, humanTimeDiff } from '@wordpress/date';
+import ConnectionIcon from '../connection-icon';
+import { ShareStatusAction } from './share-status-action';
+import { ShareStatusLabel } from './share-status-label';
+import styles from './styles.module.scss';
+
+/**
+ *
+ * ShareInfo component
+ *
+ * @param {object} props       - component props
+ * @param {object} props.share - share object
+ * @return {import('react').ReactNode} - React element
+ */
+export function ShareInfo( { share } ) {
+	const { service, external_name, profile_picture, timestamp, status, message } = share;
```

Comment:
What happens `external_name` and `profile_picture` is undefined?

Reply:
If external_name is undefined it won't be shown as with the case of linkedIN in your example, I'm not sure why linkedIn has no external name, but we might have to send down another data to fix that if for LinkedIn that's undefined all the time. If there is no profile picture, you get the basic connection icon as with LinkedIn in your case

Reply:
Should we fallback to the network logo if there is no profile picture?

Reply:
ConnectionIcon already does that, you can see it on Sid's screenshot for Linkedin